lib/collection/src/collection_manager/segments_searcher.rs

```diff
@@ -481,6 +481,7 @@ impl SegmentsSearcher {
         hw_measurement_acc: HwMeasurementAcc,
     ) -> CollectionResult<BTreeSet<PointIdType>> {
         let stopping_guard = StoppingGuard::new();
+        // cloning filter spawning task
```

Comment:
highlighting that we already clone the filter for the same reason in `read_filtered`